CHANGELOG/v1.11.0-rc.0.md

```diff
@@ -0,0 +1,389 @@
+- CAPD: Align CAPD conversion to conversion of other providers (#12481)
+- CAPD: Conditions: add V1Beta1 suffix and remove V1Beta2 suffix from condition types and reasons in CAPD v1beta2 packages (#12393)
+- CAPD: Implement v1beta2 contract in CAPD (#12409)
+- CAPD: Keep using v1beta1 condition in CAPD Docker backend (#12450)
+- CAPD: Promote v1beta2 condition in CAPD (#12362)
+- CAPD: Stop using v1beta1 status in CAPD controllers (#12438)
+- CAPD: Update example manifests (#12480)
+- Cluster: Remove deprecated index ByClusterClassName, ClusterByClusterClassClassName and ClusterClassNameField (#12269)
+- ClusterClass: Drop unnecessary fields from ClusterClass template references (#12358)
+- ClusterClass: Move infrastructure namingStrategy field in ClusterClass (#12216)
+- ClusterClass: Remove ClusterVariable.DefinitionFrom field (#12202)
+- ClusterClass: Remove DefaulterRemoveUnknownOrOmitableFields mutating webhook option (again) (#12404)
+- ClusterClass: Remove deprecated Cluster.spec.topology.rolloutAfter field (#12268)
+- ClusterClass: Remove deprecated ClusterCacheTracker and corresponding types (#12270)
+- ClusterClass: Rename deprecated ClusterClass Metadata fields to DeprecatedV1Beta1Metadata (#12273)
+- ClusterClass: Rename runtime extension fields in ClusterClass ExternalPatchDefinition (#12281)
+- ClusterClass: Restructure classRef field in Cluster.spec.topology (#12235)
+- clusterctl: Clusterctl describe defaults to v1beta2 (#12369)
+- clusterctl: Remove clusterctl alpha topology plan (#12283)
+- ClusterResourceSet: Change ClusterResourceSetBinding Bindings field from []*ResourceSetBinding to []ResourceSetBinding (#12476)
+- ClusterResourceSet: Make clusterName field in ClusterResourceSetBinding required (#12276)
+- ClusterResourceSet: Remove deprecated ClusterResourceSetBinding.DeleteBinding method (#12267)
+- Conditions: Swap condition packages (#12086)
+- Dependency: Bump to controller-runtime v0.21 / controller-tools v0.18 / k8s.io/* v0.33 / move to randfill (#12191)
+- e2e: Migrate E2E tests to v1beta2 (#12451)
+- e2e: Test/e2e: default to strict field validation & fix unknown field in ClusterClass YAML (#12501)
+- IPAM: Refactor reference types for IPAM (#12365)
+- KCP: KCP tolerates diff not leading to changes on machines (#12402)
+- KCP: Rename LastRemediationStatus.Timestamp to Time in KCP (#12452)
+- Machine: Drop unnecessary fields from Machine status.nodeRef (#12352)
+- MachineDeployment: Drop revisionHistory in MachineDeployment (#12274)
+- MachineDeployment: Remove MD spec.progressDeadlineSeconds (#12232)
+- MachineHealthCheck: Drop unnecessary fields from remediationTemplate references (#12368)
+- MachineHealthCheck: Rename MHC unhealthyConditions to unhealthyNodeConditions (#12245)
+- MachineSet: Make Template in MachineSet & Spec in MachineTemplateSpec required (#12420)
+- MULTIPLE_AREAS[API/CAPD]: Update ControlPlaneEndpoint InfraCluster contract, align CAPD to infra contracts (#12465)
```

Comment:
```suggestion
- API/CAPD: Update ControlPlaneEndpoint InfraCluster contract, align CAPD to infra contracts (#12465)
```
Same below

Should we adjust the release notes tool accordingly?

Reply:
This is still considered a manual step, because we're supposed to confirm that all the areas are relevant.

But in practice, I think all we ever do is remove the `MULTIPLE_AREAS[]` part as you suggest here, so we should consider just having the release notes tool do this.